contracts/test/SetRedemptionENA4Nov24.t.sol

```diff
@@ -314,6 +310,49 @@ contract SetRedemptionENA4Nov24Test is Test {
         }
     }
 
+    function test_RedemptionRightsCalculatedAmount_Apr1() public {
+        uint256 rewardsPerPToken = 63381137368827226;
+
+        uint256 originalReceivedTokens = 1324312 * 1e17;
+        uint256 newTokens =
+            2546753846 * 1e13 + 2546753846 * 1e13 + 20374030768 * 1e12 + 20374030768 * 1e12 + 20374030768 * 1e12;
```

Comment:
this section is getting a little hard to track - we should probably move them to variables next update.

we could also potentially re-use the body of the function.